src/pages/blog.tsx

```diff
@@ -0,0 +1,101 @@
+import { Container, Divider, Flex, Text, chakra } from '@chakra-ui/react';
+import { NextPage } from 'next';
+import Image from 'next/image';
+import React from 'react';
+import veltBusMan from '../assets/man-with-bus-background.jpg';
+import veltBoss from '../assets/velt-boss.jpg';
+import { Layout } from '../components/layout';
+
+export const Hero = chakra(Image, {
+  baseStyle: {
+    objectFit: 'cover',
+    width: '100%',
+    maxWidth: 'container.xl',
+    borderRadius: '2xl'
+  },
+  shouldForwardProp: prop =>
+    ['width', 'height', 'src', 'alt', 'fill', 'quality'].includes(prop),
+});
+
+const Blog: NextPage = () => (
+  <Layout>
+    <Container p="l">
```

Comment:
```suggestion
    <Container>
```

I don't think `l` makes sense here. Did you check mobile view?

Reply:
See @braaar's comment: https://github.com/veltolini/website/pull/34#issuecomment-1459637291